cmd/brew-test-bot.rb

```diff
@@ -583,6 +583,7 @@ def setup
       test "brew", "doctor"
       test "brew", "--env"
       test "brew", "config"
+      test "brew", "install", "xz"
```

Comment:
This should be conditional on if it's already installed, if `!ARGV.include?("--no-bottle")` and just before the first bottle is created https://github.com/kabel/homebrew-test-bot/blob/a8e82986e8ecbe70a02ddf540aaefbf7720afff9/cmd/brew-test-bot.rb#L628